widgetssdk/src/main/res/values/attrs.xml

```diff
@@ -140,6 +143,8 @@
     <attr name="gliaIconCallMinimize" format="reference" />
     <!-- The icon resId used in a chat head if an operator's profile image is absent -->
     <attr name="gliaIconPlaceholder" format="reference" />
+    <!-- The icon resId used in views to indicate operator put visitor on hold -->
```

Comment:
can it be same wording as above to make it more clear?

Reply:
Add suggestion

Reply:
`<!-- The icon resId used when visitor is put on hold by operator -->`
vs
`<!-- The icon resId used in views to indicate operator put visitor on hold -->`